2017-04-11 00:08:36,679 DEBUG org.apache.ibatis.logging.LogFactory (setImplementation:135) [RMI TCP Connection(5)-127.0.0.1] - Logging initialized using 'class org.apache.ibatis.logging.log4j.Log4jImpl' adapter.
2017-04-11 00:08:38,355 INFO  com.kovka.web.util.Initializer (contextInitialized:62) [RMI TCP Connection(5)-127.0.0.1] - -- start application -- 
2017-04-11 00:08:38,356 INFO  com.kovka.web.util.Initializer (contextInitialized:76) [RMI TCP Connection(5)-127.0.0.1] - **************************************************************************************************
2017-04-11 00:08:38,357 INFO  com.kovka.web.util.Initializer (contextInitialized:77) [RMI TCP Connection(5)-127.0.0.1] - contextPath=>F:\workaspace\github\Kovka\target\Kovka-1.0-SNAPSHOT\
2017-04-11 00:08:38,361 INFO  com.kovka.web.util.Initializer (contextInitialized:78) [RMI TCP Connection(5)-127.0.0.1] - **************************************************************************************************
2017-04-11 00:08:38,365 INFO  com.kovka.web.util.Initializer (contextInitialized:93) [RMI TCP Connection(5)-127.0.0.1] - Application 1.0 intitilize params [imagePath:.]
2017-04-11 00:08:38,366 INFO  com.kovka.web.util.Initializer (initFolders:195) [RMI TCP Connection(5)-127.0.0.1] - . folder already exist
2017-04-11 00:08:38,368 INFO  com.kovka.web.util.Initializer (initFolders:195) [RMI TCP Connection(5)-127.0.0.1] - .\merchantData folder already exist
2017-04-11 00:08:38,369 INFO  com.kovka.web.util.Initializer (initFolders:195) [RMI TCP Connection(5)-127.0.0.1] - .\merchantData\agreementDocument folder already exist
2017-04-11 00:08:38,370 INFO  com.kovka.web.util.Initializer (initFolders:195) [RMI TCP Connection(5)-127.0.0.1] - .\merchantData\company folder already exist
2017-04-11 00:08:38,535 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Log4JLogChute using logger 'VLOG'
2017-04-11 00:08:38,537 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Initializing Velocity, Calling init()...
2017-04-11 00:08:38,538 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Starting Apache Velocity v1.7 (compiled: 2010-11-19 12:14:37)
2017-04-11 00:08:38,539 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Default Properties File: org\apache\velocity\runtime\defaults\velocity.properties
2017-04-11 00:08:38,540 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Trying to use logger class org.apache.velocity.runtime.log.Log4JLogChute
2017-04-11 00:08:38,540 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Using logger class org.apache.velocity.runtime.log.Log4JLogChute
2017-04-11 00:08:38,558 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - ResourceLoader instantiated: org.apache.velocity.tools.view.WebappResourceLoader
2017-04-11 00:08:38,563 INFO  org.apache.velocity.runtime.log.Log4JLogChute (log:151) [RMI TCP Connection(5)-127.0.0.1] - WebappResourceLoader: added template path - '/WEB-INF/webapptemplate/'
2017-04-11 00:08:38,608 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - ResourceCache: initialized (class org.apache.velocity.runtime.resource.ResourceCacheImpl) with class java.util.Collections$SynchronizedMap cache map.
2017-04-11 00:08:38,612 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Stop
2017-04-11 00:08:38,617 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Define
2017-04-11 00:08:38,621 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Break
2017-04-11 00:08:38,626 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Evaluate
2017-04-11 00:08:38,629 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Literal
2017-04-11 00:08:38,633 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Macro
2017-04-11 00:08:38,637 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Parse
2017-04-11 00:08:38,641 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Include
2017-04-11 00:08:38,644 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Loaded System Directive: org.apache.velocity.runtime.directive.Foreach
2017-04-11 00:08:38,728 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Created '20' parsers.
2017-04-11 00:08:38,742 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : "velocimacro.library" is not set.  Trying default library: VM_global_library.vm
2017-04-11 00:08:38,744 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Could not load resource 'VM_global_library.vm' from ResourceLoader org.apache.velocity.tools.view.WebappResourceLoader: WebappResourceLoader: Resource 'VM_global_library.vm' not found.
2017-04-11 00:08:38,744 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : Default library not found.
2017-04-11 00:08:38,745 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : allowInline = true : VMs can be defined inline in templates
2017-04-11 00:08:38,745 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : allowInlineToOverride = false : VMs defined inline may NOT replace previous VM definitions
2017-04-11 00:08:38,746 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : allowInlineLocal = false : VMs defined inline will be global in scope if allowed.
2017-04-11 00:08:38,746 DEBUG org.apache.velocity.runtime.log.Log4JLogChute (log:168) [RMI TCP Connection(5)-127.0.0.1] - Velocimacro : autoload off : VM system will not automatically reload global library macros
2017-04-11 00:08:38,746 INFO  com.kovka.web.util.Initializer (contextInitialized:107) [RMI TCP Connection(5)-127.0.0.1] - -- application started -- 
2017-04-11 00:08:40,938 DEBUG freemarker.log._Log4jLoggerFactory$Log4jLogger (debug:45) [RMI TCP Connection(5)-127.0.0.1] - Couldn't find template in cache for "org/apache/struts2/dispatcher/error.ftl"("en_US", UTF-8, parsed); will try to load it.
2017-04-11 00:08:40,942 DEBUG freemarker.log._Log4jLoggerFactory$Log4jLogger (debug:45) [RMI TCP Connection(5)-127.0.0.1] - TemplateLoader.findTemplateSource("org/apache/struts2/dispatcher/error.ftl"): Found
2017-04-11 00:08:40,945 DEBUG freemarker.log._Log4jLoggerFactory$Log4jLogger (debug:45) [RMI TCP Connection(5)-127.0.0.1] - Loading template for "org/apache/struts2/dispatcher/error.ftl"("en_US", UTF-8, parsed) from "jar:file:/F:/workaspace/github/Kovka/target/Kovka-1.0-SNAPSHOT/WEB-INF/lib/struts2-core-2.5.2.jar!/org/apache/struts2/dispatcher/error.ftl"
2017-04-11 00:08:56,056 DEBUG freemarker.log._Log4jLoggerFactory$Log4jLogger (debug:45) [http-bio-8787-exec-7] - Key "location" was not found on instance of org.springframework.beans.factory.BeanCreationException. Introspection information for the class is: {getClass=public final native java.lang.Class java.lang.Object.getClass(), getLocalizedMessage=public java.lang.String java.lang.Throwable.getLocalizedMessage(), java.lang.Object@713a363f=freemarker.ext.beans.OverloadedMethods@6017e239, java.lang.Object@adc1f94={public java.lang.Throwable[] org.springframework.beans.factory.BeanCreationException.getRelatedCauses()=[Ljava.lang.Class;@6448831e, public java.lang.String java.lang.Throwable.getLocalizedMessage()=[Ljava.lang.Class;@7d12d3, public boolean org.springframework.beans.factory.BeanCreationException.contains(java.lang.Class)=[Ljava.lang.Class;@74ea02a4, public synchronized java.lang.Throwable java.lang.Throwable.initCause(java.lang.Throwable)=[Ljava.lang.Class;@2b33202a, public void org.springframework.beans.factory.BeanCreationException.addRelatedCause(java.lang.Throwable)=[Ljava.lang.Class;@b431621, public java.lang.String org.springframework.core.NestedRuntimeException.getMessage()=[Ljava.lang.Class;@63406610, public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause()=[Ljava.lang.Class;@479bb7fe, public synchronized java.lang.Throwable java.lang.Throwable.getCause()=[Ljava.lang.Class;@759bfd57, public java.lang.String org.springframework.beans.factory.BeanCreationException.toString()=[Ljava.lang.Class;@66fa1fdb, public void java.lang.Throwable.setStackTrace(java.lang.StackTraceElement[])=[Ljava.lang.Class;@6ea95f2, public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace()=[Ljava.lang.Class;@1c54e696, public final synchronized void java.lang.Throwable.addSuppressed(java.lang.Throwable)=[Ljava.lang.Class;@368fd10e, public java.lang.String org.springframework.beans.factory.BeanCreationException.getResourceDescription()=[Ljava.lang.Class;@bd04c3, public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed()=[Ljava.lang.Class;@522a5423, public final native java.lang.Class java.lang.Object.getClass()=[Ljava.lang.Class;@69f168a6, public int org.springframework.beans.BeansException.hashCode()=[Ljava.lang.Class;@213ed337, public boolean org.springframework.beans.BeansException.equals(java.lang.Object)=[Ljava.lang.Class;@49cd50cf, public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause()=[Ljava.lang.Class;@49469e99, public java.lang.String org.springframework.beans.factory.BeanCreationException.getBeanName()=[Ljava.lang.Class;@2b907b4f, public synchronized java.lang.Throwable java.lang.Throwable.fillInStackTrace()=[Ljava.lang.Class;@7ce0b261}, cause=java.beans.PropertyDescriptor[name=cause; propertyType=class java.lang.Throwable; readMethod=public synchronized java.lang.Throwable java.lang.Throwable.getCause()], getBeanName=public java.lang.String org.springframework.beans.factory.BeanCreationException.getBeanName(), getCause=public synchronized java.lang.Throwable java.lang.Throwable.getCause(), getResourceDescription=public java.lang.String org.springframework.beans.factory.BeanCreationException.getResourceDescription(), getStackTrace=public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace(), addSuppressed=public final synchronized void java.lang.Throwable.addSuppressed(java.lang.Throwable), hashCode=public int org.springframework.beans.BeansException.hashCode(), rootCause=java.beans.PropertyDescriptor[name=rootCause; propertyType=class java.lang.Throwable; readMethod=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause()], beanName=java.beans.PropertyDescriptor[name=beanName; propertyType=class java.lang.String; readMethod=public java.lang.String org.springframework.beans.factory.BeanCreationException.getBeanName()], getSuppressed=public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed(), suppressed=java.beans.PropertyDescriptor[name=suppressed; propertyType=class [Ljava.lang.Throwable;; readMethod=public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed()], stackTrace=java.beans.PropertyDescriptor[name=stackTrace; propertyType=class [Ljava.lang.StackTraceElement;; readMethod=public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace()], relatedCauses=java.beans.PropertyDescriptor[name=relatedCauses; propertyType=class [Ljava.lang.Throwable;; readMethod=public java.lang.Throwable[] org.springframework.beans.factory.BeanCreationException.getRelatedCauses()], getRootCause=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause(), class=java.beans.PropertyDescriptor[name=class; propertyType=class java.lang.Class; readMethod=public final native java.lang.Class java.lang.Object.getClass()], addRelatedCause=public void org.springframework.beans.factory.BeanCreationException.addRelatedCause(java.lang.Throwable), localizedMessage=java.beans.PropertyDescriptor[name=localizedMessage; propertyType=class java.lang.String; readMethod=public java.lang.String java.lang.Throwable.getLocalizedMessage()], printStackTrace=freemarker.ext.beans.OverloadedMethods@5f88ce1a, initCause=public synchronized java.lang.Throwable java.lang.Throwable.initCause(java.lang.Throwable), resourceDescription=java.beans.PropertyDescriptor[name=resourceDescription; propertyType=class java.lang.String; readMethod=public java.lang.String org.springframework.beans.factory.BeanCreationException.getResourceDescription()], mostSpecificCause=java.beans.PropertyDescriptor[name=mostSpecificCause; propertyType=class java.lang.Throwable; readMethod=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause()], getMessage=public java.lang.String org.springframework.core.NestedRuntimeException.getMessage(), message=java.beans.PropertyDescriptor[name=message; propertyType=class java.lang.String; readMethod=public java.lang.String org.springframework.core.NestedRuntimeException.getMessage()], getMostSpecificCause=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause(), contains=public boolean org.springframework.beans.factory.BeanCreationException.contains(java.lang.Class), setStackTrace=public void java.lang.Throwable.setStackTrace(java.lang.StackTraceElement[]), equals=public boolean org.springframework.beans.BeansException.equals(java.lang.Object), getRelatedCauses=public java.lang.Throwable[] org.springframework.beans.factory.BeanCreationException.getRelatedCauses(), toString=public java.lang.String org.springframework.beans.factory.BeanCreationException.toString(), fillInStackTrace=public synchronized java.lang.Throwable java.lang.Throwable.fillInStackTrace()}
2017-04-11 00:08:56,069 DEBUG freemarker.log._Log4jLoggerFactory$Log4jLogger (debug:45) [http-bio-8787-exec-7] - Key "location" was not found on instance of org.springframework.beans.factory.NoSuchBeanDefinitionException. Introspection information for the class is: {getClass=public final native java.lang.Class java.lang.Object.getClass(), getLocalizedMessage=public java.lang.String java.lang.Throwable.getLocalizedMessage(), java.lang.Object@713a363f=freemarker.ext.beans.OverloadedMethods@22473124, java.lang.Object@adc1f94={public java.lang.String java.lang.Throwable.getLocalizedMessage()=[Ljava.lang.Class;@aa56c20, public synchronized java.lang.Throwable java.lang.Throwable.initCause(java.lang.Throwable)=[Ljava.lang.Class;@732fa4af, public java.lang.Class org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanType()=[Ljava.lang.Class;@6a4e5cce, public java.lang.String org.springframework.core.NestedRuntimeException.getMessage()=[Ljava.lang.Class;@711b8eb2, public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause()=[Ljava.lang.Class;@50fe38dd, public synchronized java.lang.Throwable java.lang.Throwable.getCause()=[Ljava.lang.Class;@5e7af4b1, public int org.springframework.beans.factory.NoSuchBeanDefinitionException.getNumberOfBeansFound()=[Ljava.lang.Class;@46ca7b9c, public boolean org.springframework.core.NestedRuntimeException.contains(java.lang.Class)=[Ljava.lang.Class;@6b8903e9, public void java.lang.Throwable.setStackTrace(java.lang.StackTraceElement[])=[Ljava.lang.Class;@4a90e695, public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace()=[Ljava.lang.Class;@43c64c0a, public final synchronized void java.lang.Throwable.addSuppressed(java.lang.Throwable)=[Ljava.lang.Class;@56c5d975, public java.lang.String java.lang.Throwable.toString()=[Ljava.lang.Class;@75701e0f, public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed()=[Ljava.lang.Class;@3819ea7a, public final native java.lang.Class java.lang.Object.getClass()=[Ljava.lang.Class;@494e576f, public int org.springframework.beans.BeansException.hashCode()=[Ljava.lang.Class;@31b83142, public boolean org.springframework.beans.BeansException.equals(java.lang.Object)=[Ljava.lang.Class;@444dd11c, public java.lang.String org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanName()=[Ljava.lang.Class;@33f0a9d5, public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause()=[Ljava.lang.Class;@8be65eb, public synchronized java.lang.Throwable java.lang.Throwable.fillInStackTrace()=[Ljava.lang.Class;@787c779f}, cause=java.beans.PropertyDescriptor[name=cause; propertyType=class java.lang.Throwable; readMethod=public synchronized java.lang.Throwable java.lang.Throwable.getCause()], getBeanName=public java.lang.String org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanName(), getCause=public synchronized java.lang.Throwable java.lang.Throwable.getCause(), beanType=java.beans.PropertyDescriptor[name=beanType; propertyType=class java.lang.Class; readMethod=public java.lang.Class org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanType()], getStackTrace=public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace(), addSuppressed=public final synchronized void java.lang.Throwable.addSuppressed(java.lang.Throwable), hashCode=public int org.springframework.beans.BeansException.hashCode(), rootCause=java.beans.PropertyDescriptor[name=rootCause; propertyType=class java.lang.Throwable; readMethod=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause()], getNumberOfBeansFound=public int org.springframework.beans.factory.NoSuchBeanDefinitionException.getNumberOfBeansFound(), beanName=java.beans.PropertyDescriptor[name=beanName; propertyType=class java.lang.String; readMethod=public java.lang.String org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanName()], getSuppressed=public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed(), suppressed=java.beans.PropertyDescriptor[name=suppressed; propertyType=class [Ljava.lang.Throwable;; readMethod=public final synchronized java.lang.Throwable[] java.lang.Throwable.getSuppressed()], stackTrace=java.beans.PropertyDescriptor[name=stackTrace; propertyType=class [Ljava.lang.StackTraceElement;; readMethod=public java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace()], getRootCause=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getRootCause(), class=java.beans.PropertyDescriptor[name=class; propertyType=class java.lang.Class; readMethod=public final native java.lang.Class java.lang.Object.getClass()], getBeanType=public java.lang.Class org.springframework.beans.factory.NoSuchBeanDefinitionException.getBeanType(), localizedMessage=java.beans.PropertyDescriptor[name=localizedMessage; propertyType=class java.lang.String; readMethod=public java.lang.String java.lang.Throwable.getLocalizedMessage()], printStackTrace=freemarker.ext.beans.OverloadedMethods@52939cbd, initCause=public synchronized java.lang.Throwable java.lang.Throwable.initCause(java.lang.Throwable), mostSpecificCause=java.beans.PropertyDescriptor[name=mostSpecificCause; propertyType=class java.lang.Throwable; readMethod=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause()], getMessage=public java.lang.String org.springframework.core.NestedRuntimeException.getMessage(), message=java.beans.PropertyDescriptor[name=message; propertyType=class java.lang.String; readMethod=public java.lang.String org.springframework.core.NestedRuntimeException.getMessage()], numberOfBeansFound=java.beans.PropertyDescriptor[name=numberOfBeansFound; propertyType=int; readMethod=public int org.springframework.beans.factory.NoSuchBeanDefinitionException.getNumberOfBeansFound()], getMostSpecificCause=public java.lang.Throwable org.springframework.core.NestedRuntimeException.getMostSpecificCause(), contains=public boolean org.springframework.core.NestedRuntimeException.contains(java.lang.Class), setStackTrace=public void java.lang.Throwable.setStackTrace(java.lang.StackTraceElement[]), equals=public boolean org.springframework.beans.BeansException.equals(java.lang.Object), toString=public java.lang.String java.lang.Throwable.toString(), fillInStackTrace=public synchronized java.lang.Throwable java.lang.Throwable.fillInStackTrace()}
